CMakeLists.txt

```diff
@@ -59,7 +59,12 @@ add_compile_definitions(
 add_compile_options(/diagnostics:caret /W4 /WX /w14265 /w15038 /d1FastFail /guard:cf /Z7 /Gy /Zp8 /std:c++latest /permissive- /Zc:threadSafeInit- /Zl)
 
 set(VCLIBS_DEBUG_OPTIONS "/Od")
-set(VCLIBS_RELEASE_OPTIONS "/O2;/Os") # TRANSITION: Potentially remove /Os
+
+# TRANSITION: potentially remove /Os if it results in speed improvements (requires benchmarking!)
+# in version 19.32.31328, /Os results in binary size savings of 102K in the release DLL
+# (that's 15.5% savings)
+# so make certain that that tradeoff is measured when removing `/Os`

```

Comment:
Despite saying I was fine with this comment in person, I now hate it =) This is very close to being proper English prose with sentences - could we take it the rest of the way with some capitalization and punctuation?

Reply:
Sure, can do.